hooks/open_with_shotgun_launchapp.py

```diff
@@ -146,10 +153,15 @@ def _do_launch(self, launch_app_instance_name, engine_name, path, context):
         # first create folders based on the context - this is important because we 
         # are creating them in deferred mode, meaning that in some cases, new user sandboxes
         # maybe created at this point.
-        if context.task:
-            self.parent.tank.create_filesystem_structure("Task", context.task["id"], engine_name)
-        elif context.entity:
-            self.parent.tank.create_filesystem_structure(context.entity["type"], context.entity["id"], engine_name)
+        # This can fail with different kinds of exceptions if the filesystem schema is not configured
+        # correctly on the current projet. In this case, just continue.
+        try:
+            if context.task:
+                self.parent.tank.create_filesystem_structure("Task", context.task["id"], engine_name)
+            elif context.entity:
+                self.parent.tank.create_filesystem_structure(context.entity["type"], context.entity["id"], engine_name)
+        except Exception as e:
+            self.logger.debug("Cannot create filesystem structure (skipped): %s" % e, exc_info=True)
```

Comment:
May be a logger.warning without the traceback + logger.debug with the traceback?

Reply:
And should the project filesystem be created if the context is just for the project?

Reply:
Do you think it should? I can add it if needed.

Reply:
I think it would make sense, wouldn't it?